### DISCUSSION TITLE
New user, DoH settings

### DISCUSSION THREAD

**Discussion by Coasie** [AdguardTeam/AdGuardHome#7212] [Q&A]
I'm a new AGH user, just switched from Pihole yesterday. I have a question about DoH.

Basically wanting to know if DoH is covered by the default config, or if I need to configure the encryption settings, get a certificate etc, that all seems complicated.

My goal is just to ensure IoT devices are covered if they are using dns over https to bypass my previous pihole.

Thank you

**Comment by hl2guide** [comment]
It needs to be set by the user under `Settings` > `DNS settings` in the web interface.

e.g.

![1](https://github.com/user-attachments/assets/49722d01-8505-437b-879a-899f3f39677a)

Also on the same page check `Enable DNSSEC` (if you are using a DNSSEC-enabled resolver) and click the green `Save` button below it.

**Comment by tina-hello** [comment]
>IoT devices are covered if they are using dns over https to bypass my previous pihole

Do you mean you want to *prevent* IoT devices from using their own DoH endpoints? Then you'll need to add a list that block known DoH domains, though it won't stop unknown DoH endpoints and endpoints that doesn't rely on domain (Google DNS, Cloudflare DNS, and AdGuard DNS resolve DoH on their IP directly with valid cert)

**Reply by Coasie** [reply]
I guess that is what I mean.
If I have a wifi relay like this one that uses firmware from ewelink, I'd like to make it harder for them to do nefarious things.
https://www.amazon.com/MHCOZY-Self-Lock-Momentary-Interlock-Controller/dp/B08M3B1TZW (I have things like that, not that exact one) Same with SonOff devices, etc.
